2021-03-08 19:41:25,903 DEBUG server.py Была вызвана функция create_arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-08 19:41:26,203 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-08 19:41:44,410 INFO server.py Установлено соедение с ПК ('127.0.0.1', 61176)
2021-03-08 19:41:44,418 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61176)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:41:44,424 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615221704.410908, 'user': {'account_name': 'test1'}}
2021-03-08 19:41:44,715 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61176)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:41:44,715 INFO server.py Установлено соедение с ПК ('127.0.0.1', 61177)
2021-03-08 19:41:44,717 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61176)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:41:44,718 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615221704.7150376, 'account_name': 'test1'}
2021-03-08 19:41:44,723 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61176)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:41:44,725 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1400, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61177)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:41:44,725 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615221704.5042877, 'user': {'account_name': 'test3'}}
2021-03-08 19:41:44,988 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1400, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61177)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:41:44,989 INFO server.py Установлено соедение с ПК ('127.0.0.1', 61178)
2021-03-08 19:41:44,992 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61176)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:41:44,993 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615221704.9294326, 'user': 'test1'}
2021-03-08 19:41:45,007 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61176)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:41:45,011 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1400, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61177)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:41:45,011 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615221704.9880233, 'account_name': 'test3'}
2021-03-08 19:41:45,016 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1400, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61177)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:41:45,017 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1404, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61178)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:41:45,017 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615221704.5559967, 'user': {'account_name': 'test2'}}
2021-03-08 19:41:45,224 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1404, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61178)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:41:45,727 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1400, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61177)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:41:45,727 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615221705.170919, 'user': 'test3'}
2021-03-08 19:41:45,732 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1400, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61177)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:41:45,733 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1404, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61178)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:41:45,734 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615221705.2258856, 'account_name': 'test2'}
2021-03-08 19:41:45,737 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1404, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61178)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:41:46,241 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1404, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61178)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:41:46,241 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615221705.8454726, 'user': 'test2'}
2021-03-08 19:41:46,251 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1404, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61178)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:42:06,264 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1404, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61178)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:42:06,264 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1615221725.7860532, 'user': 'test2', 'account_name': 'test1'}
2021-03-08 19:42:06,431 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1404, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61178)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:42:18,944 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1404, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61178)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:42:18,944 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1615221738.7859166, 'user': 'test2', 'account_name': 'Radif'}
2021-03-08 19:42:19,076 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1404, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61178)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:42:27,082 INFO server.py Клиент ('127.0.0.1', 61178) отключился от сервера.
2021-03-08 19:43:02,326 INFO server.py Установлено соедение с ПК ('127.0.0.1', 61183)
2021-03-08 19:43:02,328 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61183)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:43:02,328 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615221782.3268247, 'user': {'account_name': 'Radif'}}
2021-03-08 19:43:02,429 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61183)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:43:02,933 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61183)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:43:02,933 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615221782.4296792, 'account_name': 'Radif'}
2021-03-08 19:43:02,940 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61183)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:43:03,442 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61183)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:43:03,442 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615221783.046553, 'user': 'Radif'}
2021-03-08 19:43:03,447 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61183)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:43:08,455 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61183)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:43:08,455 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1615221787.9851556, 'user': 'Radif', 'account_name': 'test3'}
2021-03-08 19:43:08,679 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61183)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:43:16,186 INFO server.py Клиент ('127.0.0.1', 61183) отключился от сервера.
2021-03-08 19:43:33,866 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1400, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61177)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 19:43:33,867 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1615221813.793106, 'user': 'test3', 'account_name': 'test1'}
2021-03-08 19:43:34,003 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1400, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61177)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 19:43:41,011 INFO server.py Клиент ('127.0.0.1', 61177) отключился от сервера.
2021-03-08 20:10:56,866 DEBUG server.py Была вызвана функция create_arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-08 20:10:57,077 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-08 20:11:19,156 INFO server.py Установлено соедение с ПК ('127.0.0.1', 61258)
2021-03-08 20:11:19,162 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61258)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:19,163 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615223479.156449, 'user': {'account_name': 'test3'}}
2021-03-08 20:11:19,341 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61258)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:11:19,342 INFO server.py Установлено соедение с ПК ('127.0.0.1', 61259)
2021-03-08 20:11:19,345 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61258)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:19,345 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615223479.3416471, 'account_name': 'test3'}
2021-03-08 20:11:19,355 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61258)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:11:19,359 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61259)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:19,359 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615223479.2234497, 'user': {'account_name': 'test1'}}
2021-03-08 20:11:19,553 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61259)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:11:19,553 INFO server.py Установлено соедение с ПК ('127.0.0.1', 61260)
2021-03-08 20:11:19,556 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61258)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:19,556 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615223479.4969065, 'user': 'test3'}
2021-03-08 20:11:19,565 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61258)>, {'response': 202, 'data_list': ['test1']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:11:19,568 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61259)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:19,568 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615223479.5534549, 'account_name': 'test1'}
2021-03-08 20:11:19,573 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61259)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:11:19,575 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1404, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61260)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:19,576 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615223479.2515252, 'user': {'account_name': 'test2'}}
2021-03-08 20:11:38,155 DEBUG server.py Была вызвана функция create_arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-08 20:11:38,306 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-08 20:11:45,709 INFO server.py Установлено соедение с ПК ('127.0.0.1', 61262)
2021-03-08 20:11:45,712 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61262)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:45,712 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615223505.7092571, 'user': {'account_name': 'test1'}}
2021-03-08 20:11:45,869 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61262)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:11:45,870 INFO server.py Установлено соедение с ПК ('127.0.0.1', 61263)
2021-03-08 20:11:45,874 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61262)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:45,874 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615223505.8694427, 'account_name': 'test1'}
2021-03-08 20:11:45,883 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61262)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:11:45,885 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1252, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61263)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:45,886 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615223505.7648363, 'user': {'account_name': 'test2'}}
2021-03-08 20:11:46,099 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1252, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61263)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:11:46,100 INFO server.py Установлено соедение с ПК ('127.0.0.1', 61264)
2021-03-08 20:11:46,104 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61262)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:46,104 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615223506.0650704, 'user': 'test1'}
2021-03-08 20:11:46,118 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61262)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:11:46,121 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1252, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61263)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:46,121 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615223506.1009839, 'account_name': 'test2'}
2021-03-08 20:11:46,124 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1252, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61263)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:11:46,126 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:46,126 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615223505.8068128, 'user': {'account_name': 'test3'}}
2021-03-08 20:11:46,329 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:11:46,832 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1252, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61263)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:46,833 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615223506.2953112, 'user': 'test2'}
2021-03-08 20:11:46,843 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1252, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61263)>, {'response': 202, 'data_list': ['test1', 'Radif']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:11:46,847 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:46,847 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615223506.32929, 'account_name': 'test3'}
2021-03-08 20:11:46,856 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:11:47,358 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:11:47,358 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615223507.1523519, 'user': 'test3'}
2021-03-08 20:11:47,364 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>, {'response': 202, 'data_list': ['test1']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:12:16,388 INFO server.py Клиент ('127.0.0.1', 61263) отключился от сервера.
2021-03-08 20:12:42,674 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:12:42,674 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615223562.255042, 'account_name': 'test3'}
2021-03-08 20:12:42,678 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:12:43,683 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:12:43,684 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615223563.2467728, 'account_name': 'test3'}
2021-03-08 20:12:43,690 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:12:44,193 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:12:44,193 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615223563.8015254, 'account_name': 'test3'}
2021-03-08 20:12:44,201 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:12:45,206 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:12:45,206 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615223564.3039575, 'account_name': 'test3'}
2021-03-08 20:12:45,213 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:12:45,714 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:12:45,714 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615223565.3600738, 'account_name': 'test3'}
2021-03-08 20:12:45,717 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61264)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:12:56,725 INFO server.py Клиент ('127.0.0.1', 61264) отключился от сервера.
2021-03-08 20:13:06,418 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61262)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:13:06,418 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1615223586.3595643, 'account_name': 'test1'}
2021-03-08 20:13:06,535 INFO server.py Клиент test1 корректно отключился от сервера.
2021-03-08 20:14:49,457 DEBUG server.py Была вызвана функция create_arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-08 20:14:49,766 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-08 20:16:39,700 INFO server.py Установлено соедение с ПК ('127.0.0.1', 61284)
2021-03-08 20:16:39,702 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1000, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61284)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:16:39,702 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615223799.700455, 'user': {'account_name': 'Radif'}}
2021-03-08 20:16:39,916 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1000, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61284)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:16:40,426 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1000, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61284)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:16:40,426 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615223799.9113111, 'account_name': 'Radif'}
2021-03-08 20:16:40,430 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1000, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61284)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:16:40,935 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1000, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61284)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:16:40,935 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615223800.7625139, 'user': 'Radif'}
2021-03-08 20:16:40,943 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1000, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 61284)>, {'response': 202, 'data_list': ['test3']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:16:47,450 INFO server.py Клиент ('127.0.0.1', 61284) отключился от сервера.
2021-03-08 20:20:06,178 DEBUG server.py Была вызвана функция create_arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-08 20:20:06,369 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-08 20:20:14,157 INFO server.py Установлено соедение с ПК ('127.0.0.1', 49760)
2021-03-08 20:20:14,163 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49760)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:20:14,163 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615224014.1570373, 'user': {'account_name': 'test2'}}
2021-03-08 20:20:14,338 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49760)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:20:14,338 INFO server.py Установлено соедение с ПК ('127.0.0.1', 49761)
2021-03-08 20:20:14,341 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49760)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:20:14,341 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615224014.338201, 'account_name': 'test2'}
2021-03-08 20:20:14,351 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49760)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:20:14,354 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49761)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:20:14,354 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615224014.1961043, 'user': {'account_name': 'test3'}}
2021-03-08 20:20:14,550 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49761)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:20:14,550 INFO server.py Установлено соедение с ПК ('127.0.0.1', 49762)
2021-03-08 20:20:14,556 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49760)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:20:14,556 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615224014.4797153, 'user': 'test2'}
2021-03-08 20:20:14,581 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49760)>, {'response': 202, 'data_list': ['test1', 'Radif']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:20:14,584 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49761)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:20:14,585 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615224014.550325, 'account_name': 'test3'}
2021-03-08 20:20:14,593 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49761)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:20:14,596 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49762)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:20:14,596 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615224014.224813, 'user': {'account_name': 'test1'}}
2021-03-08 20:20:27,411 DEBUG server.py Была вызвана функция create_arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-08 20:20:27,550 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-08 20:20:49,970 DEBUG server.py Была вызвана функция create_arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-08 20:20:50,164 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-08 20:21:04,427 DEBUG server.py Была вызвана функция create_arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-08 20:21:04,608 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-08 20:21:14,315 INFO server.py Установлено соедение с ПК ('127.0.0.1', 49789)
2021-03-08 20:21:14,323 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49789)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:21:14,323 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615224074.3151948, 'user': {'account_name': 'test1'}}
2021-03-08 20:21:14,485 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49789)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:21:14,486 INFO server.py Установлено соедение с ПК ('127.0.0.1', 49790)
2021-03-08 20:21:14,489 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49789)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:21:14,489 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615224074.4864547, 'account_name': 'test1'}
2021-03-08 20:21:14,498 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49789)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:21:14,501 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49790)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:21:14,502 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615224074.334179, 'user': {'account_name': 'test2'}}
2021-03-08 20:21:14,676 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49790)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:21:14,677 INFO server.py Установлено соедение с ПК ('127.0.0.1', 49791)
2021-03-08 20:21:14,680 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49789)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:21:14,680 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615224074.628108, 'user': 'test1'}
2021-03-08 20:21:14,694 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49789)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:21:14,698 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1408, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49791)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:21:14,699 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615224074.3946164, 'user': {'account_name': 'test3'}}
2021-03-08 20:21:14,792 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1408, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49791)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:21:15,295 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49790)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:21:15,295 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615224074.6787992, 'account_name': 'test2'}
2021-03-08 20:21:15,301 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49790)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:21:15,304 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1408, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49791)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:21:15,305 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615224074.7925563, 'account_name': 'test3'}
2021-03-08 20:21:15,313 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1408, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49791)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:21:15,817 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49790)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:21:15,817 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615224075.532179, 'user': 'test2'}
2021-03-08 20:21:15,826 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49790)>, {'response': 202, 'data_list': ['test1', 'Radif']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:21:15,829 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1408, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49791)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:21:15,829 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615224075.548658, 'user': 'test3'}
2021-03-08 20:21:15,839 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1408, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49791)>, {'response': 202, 'data_list': ['test1']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:21:24,343 INFO server.py Клиент ('127.0.0.1', 49790) отключился от сервера.
2021-03-08 20:21:32,983 INFO server.py Клиент ('127.0.0.1', 49791) отключился от сервера.
2021-03-08 20:21:37,158 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49789)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:21:37,158 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1615224096.7037117, 'account_name': 'test1'}
2021-03-08 20:21:37,334 INFO server.py Клиент test1 корректно отключился от сервера.
2021-03-08 20:22:18,398 DEBUG server.py Была вызвана функция create_arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-08 20:22:18,638 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-08 20:22:34,331 INFO server.py Установлено соедение с ПК ('127.0.0.1', 49798)
2021-03-08 20:22:34,332 INFO server.py Установлено соедение с ПК ('127.0.0.1', 49799)
2021-03-08 20:22:34,338 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49798)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:22:34,339 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615224154.3312886, 'user': {'account_name': 'test3'}}
2021-03-08 20:22:34,438 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49798)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:22:34,441 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:22:34,441 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615224154.3312886, 'user': {'account_name': 'test2'}}
2021-03-08 20:22:34,552 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:22:34,552 INFO server.py Установлено соедение с ПК ('127.0.0.1', 49800)
2021-03-08 20:22:34,553 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49798)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:22:34,553 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615224154.438014, 'account_name': 'test3'}
2021-03-08 20:22:34,558 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49798)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:22:34,560 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:22:34,560 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615224154.5528076, 'account_name': 'test2'}
2021-03-08 20:22:34,564 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:22:34,566 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:22:34,566 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615224154.346279, 'user': {'account_name': 'test1'}}
2021-03-08 20:22:34,792 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:22:35,295 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49798)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:22:35,295 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615224154.743112, 'user': 'test3'}
2021-03-08 20:22:35,305 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49798)>, {'response': 202, 'data_list': ['test1']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:22:35,308 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:22:35,308 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615224154.810647, 'user': 'test2'}
2021-03-08 20:22:35,315 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>, {'response': 202, 'data_list': ['test1', 'Radif']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:22:35,318 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:22:35,318 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615224154.792194, 'account_name': 'test1'}
2021-03-08 20:22:35,323 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:22:35,826 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:22:35,826 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615224155.5672479, 'user': 'test1'}
2021-03-08 20:22:35,832 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:23:00,348 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:23:00,348 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1615224180.3367946, 'mess_text': 'hi\n'}
2021-03-08 20:23:00,527 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:23:00,531 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>, {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1615224180.3367946, 'mess_text': 'hi\n'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_message
2021-03-08 20:23:00,531 INFO server.py Отправлено сообщение пользователю test1 от пользователя test2.
2021-03-08 20:23:12,538 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:23:12,538 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1615224192.4963794, 'mess_text': 'hi\n'}
2021-03-08 20:23:12,664 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:23:12,670 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>, {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1615224192.4963794, 'mess_text': 'hi\n'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_message
2021-03-08 20:23:12,670 INFO server.py Отправлено сообщение пользователю test1 от пользователя test2.
2021-03-08 20:23:14,676 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:23:14,676 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1615224194.3002164, 'user': 'test1', 'account_name': 'test2'}
2021-03-08 20:23:14,860 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:23:25,872 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:23:25,872 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1615224205.792224, 'mess_text': 'and you'}
2021-03-08 20:23:26,031 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:23:26,035 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>, {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1615224205.792224, 'mess_text': 'and you'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_message
2021-03-08 20:23:26,036 INFO server.py Отправлено сообщение пользователю test2 от пользователя test1.
2021-03-08 20:23:37,045 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:23:37,045 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'Radif', 'time': 1615224216.7598825, 'mess_text': 'hi'}
2021-03-08 20:23:37,048 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>, {'response': 400, 'error': 'Пользователь не зарегистрирован на сервере.'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:23:53,883 INFO server.py Установлено соедение с ПК ('127.0.0.1', 49805)
2021-03-08 20:23:54,385 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49805)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:23:54,385 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615224233.883896, 'user': {'account_name': 'Radif'}}
2021-03-08 20:23:54,636 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49805)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:23:55,137 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49805)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:23:55,137 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615224234.6360211, 'account_name': 'Radif'}
2021-03-08 20:23:55,140 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49805)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:23:55,643 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49805)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:23:55,643 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615224235.3378162, 'user': 'Radif'}
2021-03-08 20:23:55,652 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49805)>, {'response': 202, 'data_list': ['test3']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:25:10,722 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:25:10,722 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'Radif', 'time': 1615224310.6383963, 'mess_text': 'look\n'}
2021-03-08 20:25:11,251 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:25:11,256 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49805)>, {'action': 'message', 'from': 'test2', 'to': 'Radif', 'time': 1615224310.6383963, 'mess_text': 'look\n'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_message
2021-03-08 20:25:11,256 INFO server.py Отправлено сообщение пользователю Radif от пользователя test2.
2021-03-08 20:25:29,272 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49805)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:25:29,273 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1615224329.2347803, 'user': 'Radif', 'account_name': 'test2'}
2021-03-08 20:25:29,498 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49805)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:25:42,509 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49805)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:25:42,510 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'Radif', 'to': 'test2', 'time': 1615224342.0861137, 'mess_text': 'radif.ru\n'}
2021-03-08 20:25:42,667 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49805)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:25:42,671 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>, {'action': 'message', 'from': 'Radif', 'to': 'test2', 'time': 1615224342.0861137, 'mess_text': 'radif.ru\n'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_message
2021-03-08 20:25:42,671 INFO server.py Отправлено сообщение пользователю test2 от пользователя Radif.
2021-03-08 20:28:09,811 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:28:09,811 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1615224489.4054086, 'user': 'test1', 'account_name': 'Radif'}
2021-03-08 20:28:10,274 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:28:24,786 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:28:24,786 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1615224504.5087373, 'mess_text': '+\n'}
2021-03-08 20:28:24,996 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:28:25,001 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>, {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1615224504.5087373, 'mess_text': '+\n'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_message
2021-03-08 20:28:25,002 INFO server.py Отправлено сообщение пользователю test2 от пользователя test1.
2021-03-08 20:29:09,032 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:29:09,032 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'Radif', 'time': 1615224548.7005906, 'mess_text': '  \n\n\n\n\n  '}
2021-03-08 20:29:09,623 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:29:09,627 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49805)>, {'action': 'message', 'from': 'test1', 'to': 'Radif', 'time': 1615224548.7005906, 'mess_text': '  \n\n\n\n\n  '}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_message
2021-03-08 20:29:09,627 INFO server.py Отправлено сообщение пользователю Radif от пользователя test1.
2021-03-08 20:29:17,134 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49805)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:29:17,134 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1615224557.0966194, 'user': 'Radif', 'account_name': 'test1'}
2021-03-08 20:29:17,331 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49805)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-08 20:35:00,111 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1396, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49799)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:35:00,111 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1615224899.6241271, 'account_name': 'test2'}
2021-03-08 20:35:00,693 INFO server.py Клиент test2 корректно отключился от сервера.
2021-03-08 20:35:09,202 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-08 20:35:09,202 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1615224909.1362202, 'mess_text': '-\n'}
2021-03-08 20:35:09,206 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 49800)>, {'response': 400, 'error': 'Пользователь не зарегистрирован на сервере.'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-09 16:07:04,006 DEBUG server.py Была вызвана функция create_arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-09 16:07:04,591 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: 127.0.0.1. Если адрес не указан, принимаются соединения с любых адресов.
2021-03-09 16:07:05,193 INFO server.py Установлено соедение с ПК ('127.0.0.1', 51396)
2021-03-09 16:07:05,255 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51396)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-09 16:07:05,256 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615295224.8779268, 'user': {'account_name': 'test3'}}
2021-03-09 16:07:05,529 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51396)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-09 16:07:05,529 INFO server.py Установлено соедение с ПК ('127.0.0.1', 51397)
2021-03-09 16:07:05,534 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51396)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-09 16:07:05,534 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615295225.5293458, 'account_name': 'test3'}
2021-03-09 16:07:05,544 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51396)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-09 16:07:05,548 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1268, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51397)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-09 16:07:05,548 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615295224.8779268, 'user': {'account_name': 'test1'}}
2021-03-09 16:07:05,944 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1268, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51397)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-09 16:07:05,944 INFO server.py Установлено соедение с ПК ('127.0.0.1', 51398)
2021-03-09 16:07:05,949 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51396)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-09 16:07:05,949 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615295225.8145149, 'user': 'test3'}
2021-03-09 16:07:05,964 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51396)>, {'response': 202, 'data_list': ['test1']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-09 16:07:05,965 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1268, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51397)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-09 16:07:05,966 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615295225.9425066, 'account_name': 'test1'}
2021-03-09 16:07:05,969 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1268, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51397)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-09 16:07:05,971 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51398)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-09 16:07:05,971 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615295224.8909204, 'user': {'account_name': 'test2'}}
2021-03-09 16:07:06,229 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51398)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-09 16:07:06,730 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1268, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51397)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-09 16:07:06,730 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615295226.230332, 'user': 'test1'}
2021-03-09 16:07:06,735 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1268, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51397)>, {'response': 202, 'data_list': ['test2', 'Radif']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-09 16:07:06,736 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51398)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-09 16:07:06,737 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615295226.2293324, 'account_name': 'test2'}
2021-03-09 16:07:06,740 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51398)>, {'response': 202, 'data_list': ['test_user_1', 'test_user_2', 'test2', 'test1', 'test3', 'Radif', 'test4', 'test6']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-09 16:07:07,242 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51398)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-09 16:07:07,242 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615295226.9469914, 'user': 'test2'}
2021-03-09 16:07:07,247 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51398)>, {'response': 202, 'data_list': ['test1', 'Radif']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-09 16:07:23,266 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51398)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-09 16:07:23,267 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1615295243.0472863, 'mess_text': 'turtu\n'}
2021-03-09 16:07:23,427 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51398)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-09 16:07:23,436 DEBUG server.py Была вызвана функция send_message c параметрами (<socket.socket fd=1268, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51397)>, {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1615295243.0472863, 'mess_text': 'turtu\n'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_message
2021-03-09 16:07:23,436 INFO server.py Отправлено сообщение пользователю test1 от пользователя test2.
2021-03-09 16:07:33,448 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1268, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51397)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-09 16:07:33,449 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1615295253.1072526, 'account_name': 'test1'}
2021-03-09 16:07:33,568 INFO server.py Клиент test1 корректно отключился от сервера.
2021-03-09 16:07:34,072 DEBUG server.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 51398)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-09 16:07:34,072 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1615295253.9127076, 'account_name': 'test2'}
2021-03-09 16:07:34,266 INFO server.py Клиент test2 корректно отключился от сервера.
2021-03-10 15:51:20,423 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 15:51:20,429 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2021-03-10 15:51:20,443 DEBUG server.py Аргументы успешно загружены.
2021-03-10 15:51:20,444 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 15:51:20,671 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 15:52:16,619 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 15:52:16,621 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2021-03-10 15:52:16,633 DEBUG server.py Аргументы успешно загружены.
2021-03-10 15:52:16,634 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 15:52:17,246 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 15:54:04,755 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 15:54:04,755 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2021-03-10 15:54:04,767 DEBUG server.py Аргументы успешно загружены.
2021-03-10 15:54:04,768 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 15:54:05,035 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 15:56:02,485 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 15:56:02,544 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2021-03-10 15:56:02,558 DEBUG server.py Аргументы успешно загружены.
2021-03-10 15:56:02,560 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 15:56:02,819 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 15:56:19,425 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53931)
2021-03-10 15:56:19,679 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53932)
2021-03-10 15:56:19,683 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1280, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53931)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 15:56:19,684 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615380979.4500303, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA7BKn84ehzrpBhCqWtF+X\nbX2WFcuURuPz4847Y0DSmQNGGp0d396Il6zyEXy6wJ6c7nPiPrhw8o6+FWmvbxsG\nLxqQu3d1tUre/VRaYKuX+l1LWXm0jAXSPfqva1wENgAB1itE9uEENR0/63JOAFFa\no+B/ln22z+BfBDOuJSEEkcDwmbx74W3IiaXV3HA4o7WJo31L4feb1Iw79uzOubYE\n9DCb/m0hv0q1HeQWuNVrnw3BSQS/dHXSYxKnYfyJPQT0DBbpp5NhUmMJxK87kREr\n2AYatB4Knh7py2liqOinRSqHAntUv6cUtd1QzokeC4WNOWP7mNNRn3D5E8PFaP/W\nKQIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 15:56:19,684 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA7BKn84ehzrpBhCqWtF+X\nbX2WFcuURuPz4847Y0DSmQNGGp0d396Il6zyEXy6wJ6c7nPiPrhw8o6+FWmvbxsG\nLxqQu3d1tUre/VRaYKuX+l1LWXm0jAXSPfqva1wENgAB1itE9uEENR0/63JOAFFa\no+B/ln22z+BfBDOuJSEEkcDwmbx74W3IiaXV3HA4o7WJo31L4feb1Iw79uzOubYE\n9DCb/m0hv0q1HeQWuNVrnw3BSQS/dHXSYxKnYfyJPQT0DBbpp5NhUmMJxK87kREr\n2AYatB4Knh7py2liqOinRSqHAntUv6cUtd1QzokeC4WNOWP7mNNRn3D5E8PFaP/W\nKQIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:04:57,411 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:04:57,411 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2021-03-10 16:04:57,423 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:04:57,425 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:04:57,791 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:05:01,109 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 16:06:06,290 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:06:06,290 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2021-03-10 16:06:06,303 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:06:06,303 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:06:07,678 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:06:18,405 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53956)
2021-03-10 16:06:18,502 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53957)
2021-03-10 16:06:18,512 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1272, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53956)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:06:18,512 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615381578.4377487, 'user': {'account_name': 'test3', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA35NgXpKoG3Iz+3h5OHcL\n6bC5+HRchqrVhqlaOyS03vzMlRpHKteeGIr2kOt1CNhxqggQChzXA8SZErBu67Ur\nzEcgw8IHN6yVzr9W18E6olu1DuRQKzHY7Vhkv0uWS19Jnp9EU0PmR0sxBx7ZcWkt\n6WnMWBfsxBSMoDMUf5/b1BPOlTfbZ4TuY+jeoruM4ngFSmuT8T+6lfux4CkrdAn+\nCdleVV5el8ntT6SMdUDswLsHxZYxNmECSFL8fwbOHDqsu7H2RiYR+fMSsSORON7Y\nFMsBWRmzJ4bJ+/SNkg2tt7GqQb99fRCJt9x9OO3Bl2MQ9Ha5ah9QgUJ9twmFBAne\nyQIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 16:06:18,513 DEBUG core.py Start auth process for {'account_name': 'test3', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA35NgXpKoG3Iz+3h5OHcL\n6bC5+HRchqrVhqlaOyS03vzMlRpHKteeGIr2kOt1CNhxqggQChzXA8SZErBu67Ur\nzEcgw8IHN6yVzr9W18E6olu1DuRQKzHY7Vhkv0uWS19Jnp9EU0PmR0sxBx7ZcWkt\n6WnMWBfsxBSMoDMUf5/b1BPOlTfbZ4TuY+jeoruM4ngFSmuT8T+6lfux4CkrdAn+\nCdleVV5el8ntT6SMdUDswLsHxZYxNmECSFL8fwbOHDqsu7H2RiYR+fMSsSORON7Y\nFMsBWRmzJ4bJ+/SNkg2tt7GqQb99fRCJt9x9OO3Bl2MQ9Ha5ah9QgUJ9twmFBAne\nyQIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:06:18,525 DEBUG core.py Unknown username, sending {'response': 400, 'error': 'Пользователь не зарегистрирован.'}
2021-03-10 16:06:18,533 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1272, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53956)>, {'response': 400, 'error': 'Пользователь не зарегистрирован.'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:06:18,534 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53958)
2021-03-10 16:06:19,037 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1288, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53957)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:06:19,037 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615381578.5506775, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA7BKn84ehzrpBhCqWtF+X\nbX2WFcuURuPz4847Y0DSmQNGGp0d396Il6zyEXy6wJ6c7nPiPrhw8o6+FWmvbxsG\nLxqQu3d1tUre/VRaYKuX+l1LWXm0jAXSPfqva1wENgAB1itE9uEENR0/63JOAFFa\no+B/ln22z+BfBDOuJSEEkcDwmbx74W3IiaXV3HA4o7WJo31L4feb1Iw79uzOubYE\n9DCb/m0hv0q1HeQWuNVrnw3BSQS/dHXSYxKnYfyJPQT0DBbpp5NhUmMJxK87kREr\n2AYatB4Knh7py2liqOinRSqHAntUv6cUtd1QzokeC4WNOWP7mNNRn3D5E8PFaP/W\nKQIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 16:06:19,038 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA7BKn84ehzrpBhCqWtF+X\nbX2WFcuURuPz4847Y0DSmQNGGp0d396Il6zyEXy6wJ6c7nPiPrhw8o6+FWmvbxsG\nLxqQu3d1tUre/VRaYKuX+l1LWXm0jAXSPfqva1wENgAB1itE9uEENR0/63JOAFFa\no+B/ln22z+BfBDOuJSEEkcDwmbx74W3IiaXV3HA4o7WJo31L4feb1Iw79uzOubYE\n9DCb/m0hv0q1HeQWuNVrnw3BSQS/dHXSYxKnYfyJPQT0DBbpp5NhUmMJxK87kREr\n2AYatB4Knh7py2liqOinRSqHAntUv6cUtd1QzokeC4WNOWP7mNNRn3D5E8PFaP/W\nKQIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:06:19,044 DEBUG core.py Unknown username, sending {'response': 400, 'error': 'Пользователь не зарегистрирован.'}
2021-03-10 16:06:19,051 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1288, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53957)>, {'response': 400, 'error': 'Пользователь не зарегистрирован.'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:06:19,054 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1272, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53958)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:06:19,055 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615381578.5566735, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAkCloVMkGdxuoqW9r4GhP\np2rX5T0GDTIzNLc4fX38sGK9jUlQRvcbYrObBLfSFUQGy1fSQu+POR61Ik99BUpR\nXwzRVnnLXCFacoQDVkBZjNH1X0uff/Px7hOhZ1yXvM9aYercGvkC71oubZfJApDk\ny/0axjjxXToKUYLvwtXC8/s/wuToNKg9oI/EUtuIvMe/1pa7KBj322B92X+zijvD\nFW+YENwoyeykVARchsGZmtREKWJ/wTowHS8RY8nYhnoV6pW159H6RvPT6fyTf/GO\nGrMnmFkxDhs03eZQEElpjvItiJOysn/uirZw0kMlENTIO0kXhMq2t+nJwrL7YYpN\n6QIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 16:06:19,055 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAkCloVMkGdxuoqW9r4GhP\np2rX5T0GDTIzNLc4fX38sGK9jUlQRvcbYrObBLfSFUQGy1fSQu+POR61Ik99BUpR\nXwzRVnnLXCFacoQDVkBZjNH1X0uff/Px7hOhZ1yXvM9aYercGvkC71oubZfJApDk\ny/0axjjxXToKUYLvwtXC8/s/wuToNKg9oI/EUtuIvMe/1pa7KBj322B92X+zijvD\nFW+YENwoyeykVARchsGZmtREKWJ/wTowHS8RY8nYhnoV6pW159H6RvPT6fyTf/GO\nGrMnmFkxDhs03eZQEElpjvItiJOysn/uirZw0kMlENTIO0kXhMq2t+nJwrL7YYpN\n6QIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:06:19,062 DEBUG core.py Unknown username, sending {'response': 400, 'error': 'Пользователь не зарегистрирован.'}
2021-03-10 16:06:19,070 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1272, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53958)>, {'response': 400, 'error': 'Пользователь не зарегистрирован.'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:07:36,335 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53961)
2021-03-10 16:07:36,405 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53962)
2021-03-10 16:07:36,407 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1688, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53961)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:07:36,407 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615381656.366776, 'user': {'account_name': 'test3', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA35NgXpKoG3Iz+3h5OHcL\n6bC5+HRchqrVhqlaOyS03vzMlRpHKteeGIr2kOt1CNhxqggQChzXA8SZErBu67Ur\nzEcgw8IHN6yVzr9W18E6olu1DuRQKzHY7Vhkv0uWS19Jnp9EU0PmR0sxBx7ZcWkt\n6WnMWBfsxBSMoDMUf5/b1BPOlTfbZ4TuY+jeoruM4ngFSmuT8T+6lfux4CkrdAn+\nCdleVV5el8ntT6SMdUDswLsHxZYxNmECSFL8fwbOHDqsu7H2RiYR+fMSsSORON7Y\nFMsBWRmzJ4bJ+/SNkg2tt7GqQb99fRCJt9x9OO3Bl2MQ9Ha5ah9QgUJ9twmFBAne\nyQIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 16:07:36,407 DEBUG core.py Start auth process for {'account_name': 'test3', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA35NgXpKoG3Iz+3h5OHcL\n6bC5+HRchqrVhqlaOyS03vzMlRpHKteeGIr2kOt1CNhxqggQChzXA8SZErBu67Ur\nzEcgw8IHN6yVzr9W18E6olu1DuRQKzHY7Vhkv0uWS19Jnp9EU0PmR0sxBx7ZcWkt\n6WnMWBfsxBSMoDMUf5/b1BPOlTfbZ4TuY+jeoruM4ngFSmuT8T+6lfux4CkrdAn+\nCdleVV5el8ntT6SMdUDswLsHxZYxNmECSFL8fwbOHDqsu7H2RiYR+fMSsSORON7Y\nFMsBWRmzJ4bJ+/SNkg2tt7GqQb99fRCJt9x9OO3Bl2MQ9Ha5ah9QgUJ9twmFBAne\nyQIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:07:36,411 DEBUG core.py Correct username, starting passwd check.
2021-03-10 16:07:36,413 DEBUG core.py Auth message = {'response': 511, 'bin': '28dd32fefaa5984968f35c844cb0bb12f33fe71b6e6be5d8a46615626a760890a785d51a37b48706d03be688bc6f2fba50927c13568cc2b4e7e34ab87fce7e9d'}
2021-03-10 16:07:36,416 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1688, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53961)>, {'response': 511, 'bin': '28dd32fefaa5984968f35c844cb0bb12f33fe71b6e6be5d8a46615626a760890a785d51a37b48706d03be688bc6f2fba50927c13568cc2b4e7e34ab87fce7e9d'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:07:36,419 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1688, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53961)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:07:36,422 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1688, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53961)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:07:36,555 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53963)
2021-03-10 16:07:36,559 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1688, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53961)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:07:36,559 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615381656.4224942, 'account_name': 'test3'}
2021-03-10 16:07:36,570 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1688, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53961)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3', 'Radif']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:07:36,574 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1660, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53962)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:07:36,574 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615381656.421495, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA7BKn84ehzrpBhCqWtF+X\nbX2WFcuURuPz4847Y0DSmQNGGp0d396Il6zyEXy6wJ6c7nPiPrhw8o6+FWmvbxsG\nLxqQu3d1tUre/VRaYKuX+l1LWXm0jAXSPfqva1wENgAB1itE9uEENR0/63JOAFFa\no+B/ln22z+BfBDOuJSEEkcDwmbx74W3IiaXV3HA4o7WJo31L4feb1Iw79uzOubYE\n9DCb/m0hv0q1HeQWuNVrnw3BSQS/dHXSYxKnYfyJPQT0DBbpp5NhUmMJxK87kREr\n2AYatB4Knh7py2liqOinRSqHAntUv6cUtd1QzokeC4WNOWP7mNNRn3D5E8PFaP/W\nKQIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 16:07:36,575 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA7BKn84ehzrpBhCqWtF+X\nbX2WFcuURuPz4847Y0DSmQNGGp0d396Il6zyEXy6wJ6c7nPiPrhw8o6+FWmvbxsG\nLxqQu3d1tUre/VRaYKuX+l1LWXm0jAXSPfqva1wENgAB1itE9uEENR0/63JOAFFa\no+B/ln22z+BfBDOuJSEEkcDwmbx74W3IiaXV3HA4o7WJo31L4feb1Iw79uzOubYE\n9DCb/m0hv0q1HeQWuNVrnw3BSQS/dHXSYxKnYfyJPQT0DBbpp5NhUmMJxK87kREr\n2AYatB4Knh7py2liqOinRSqHAntUv6cUtd1QzokeC4WNOWP7mNNRn3D5E8PFaP/W\nKQIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:07:36,584 DEBUG core.py Correct username, starting passwd check.
2021-03-10 16:07:36,589 DEBUG core.py Auth message = {'response': 511, 'bin': '580e24dcba79f89ccec735c1ec51d089d1818ba806ff57dbc8fc907ac939d3a61966f1b85c769846b0dfd721abe6988960c1b0c6e2ac6e45946f3c7811750736'}
2021-03-10 16:07:36,597 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1660, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53962)>, {'response': 511, 'bin': '580e24dcba79f89ccec735c1ec51d089d1818ba806ff57dbc8fc907ac939d3a61966f1b85c769846b0dfd721abe6988960c1b0c6e2ac6e45946f3c7811750736'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:07:36,601 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1660, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53962)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:07:36,603 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1660, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53962)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:07:36,832 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1772, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53963)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:07:36,832 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615381656.4384847, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAkCloVMkGdxuoqW9r4GhP\np2rX5T0GDTIzNLc4fX38sGK9jUlQRvcbYrObBLfSFUQGy1fSQu+POR61Ik99BUpR\nXwzRVnnLXCFacoQDVkBZjNH1X0uff/Px7hOhZ1yXvM9aYercGvkC71oubZfJApDk\ny/0axjjxXToKUYLvwtXC8/s/wuToNKg9oI/EUtuIvMe/1pa7KBj322B92X+zijvD\nFW+YENwoyeykVARchsGZmtREKWJ/wTowHS8RY8nYhnoV6pW159H6RvPT6fyTf/GO\nGrMnmFkxDhs03eZQEElpjvItiJOysn/uirZw0kMlENTIO0kXhMq2t+nJwrL7YYpN\n6QIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 16:07:36,833 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAkCloVMkGdxuoqW9r4GhP\np2rX5T0GDTIzNLc4fX38sGK9jUlQRvcbYrObBLfSFUQGy1fSQu+POR61Ik99BUpR\nXwzRVnnLXCFacoQDVkBZjNH1X0uff/Px7hOhZ1yXvM9aYercGvkC71oubZfJApDk\ny/0axjjxXToKUYLvwtXC8/s/wuToNKg9oI/EUtuIvMe/1pa7KBj322B92X+zijvD\nFW+YENwoyeykVARchsGZmtREKWJ/wTowHS8RY8nYhnoV6pW159H6RvPT6fyTf/GO\nGrMnmFkxDhs03eZQEElpjvItiJOysn/uirZw0kMlENTIO0kXhMq2t+nJwrL7YYpN\n6QIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:07:36,839 DEBUG core.py Correct username, starting passwd check.
2021-03-10 16:07:36,841 DEBUG core.py Auth message = {'response': 511, 'bin': 'c3ea917de08101cf296cbe1d3d617c08de7dd7c724e3b1b3ff7e1f1a42e296f57b0343ec33b23320a09c5287feb0c9d37eee1bf056a24276ab93dfc792ca9855'}
2021-03-10 16:07:36,848 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1772, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53963)>, {'response': 511, 'bin': 'c3ea917de08101cf296cbe1d3d617c08de7dd7c724e3b1b3ff7e1f1a42e296f57b0343ec33b23320a09c5287feb0c9d37eee1bf056a24276ab93dfc792ca9855'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:07:36,851 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1772, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53963)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:07:36,856 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1772, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53963)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:07:51,123 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:07:51,124 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2021-03-10 16:07:51,136 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:07:51,138 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:07:51,288 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:08:06,486 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53965)
2021-03-10 16:08:06,487 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53966)
2021-03-10 16:08:06,502 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53967)
2021-03-10 16:08:07,009 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53965)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:08:07,009 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615381686.5232644, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAkCloVMkGdxuoqW9r4GhP\np2rX5T0GDTIzNLc4fX38sGK9jUlQRvcbYrObBLfSFUQGy1fSQu+POR61Ik99BUpR\nXwzRVnnLXCFacoQDVkBZjNH1X0uff/Px7hOhZ1yXvM9aYercGvkC71oubZfJApDk\ny/0axjjxXToKUYLvwtXC8/s/wuToNKg9oI/EUtuIvMe/1pa7KBj322B92X+zijvD\nFW+YENwoyeykVARchsGZmtREKWJ/wTowHS8RY8nYhnoV6pW159H6RvPT6fyTf/GO\nGrMnmFkxDhs03eZQEElpjvItiJOysn/uirZw0kMlENTIO0kXhMq2t+nJwrL7YYpN\n6QIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 16:08:07,009 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAkCloVMkGdxuoqW9r4GhP\np2rX5T0GDTIzNLc4fX38sGK9jUlQRvcbYrObBLfSFUQGy1fSQu+POR61Ik99BUpR\nXwzRVnnLXCFacoQDVkBZjNH1X0uff/Px7hOhZ1yXvM9aYercGvkC71oubZfJApDk\ny/0axjjxXToKUYLvwtXC8/s/wuToNKg9oI/EUtuIvMe/1pa7KBj322B92X+zijvD\nFW+YENwoyeykVARchsGZmtREKWJ/wTowHS8RY8nYhnoV6pW159H6RvPT6fyTf/GO\nGrMnmFkxDhs03eZQEElpjvItiJOysn/uirZw0kMlENTIO0kXhMq2t+nJwrL7YYpN\n6QIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:08:07,016 DEBUG core.py Correct username, starting passwd check.
2021-03-10 16:08:07,019 DEBUG core.py Auth message = {'response': 511, 'bin': 'd3964b475350da6e85360b260ea429354692bb8c704a0eb5f68f785d735f4035de9ec8bec21d0ac388d69b45797bac5d5abf31ba20dca1c6b80aee5849b0b983'}
2021-03-10 16:08:07,023 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53965)>, {'response': 511, 'bin': 'd3964b475350da6e85360b260ea429354692bb8c704a0eb5f68f785d735f4035de9ec8bec21d0ac388d69b45797bac5d5abf31ba20dca1c6b80aee5849b0b983'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:08:07,025 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53965)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:08:07,028 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53965)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:08:07,208 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53966)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:08:07,209 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615381686.524266, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA7BKn84ehzrpBhCqWtF+X\nbX2WFcuURuPz4847Y0DSmQNGGp0d396Il6zyEXy6wJ6c7nPiPrhw8o6+FWmvbxsG\nLxqQu3d1tUre/VRaYKuX+l1LWXm0jAXSPfqva1wENgAB1itE9uEENR0/63JOAFFa\no+B/ln22z+BfBDOuJSEEkcDwmbx74W3IiaXV3HA4o7WJo31L4feb1Iw79uzOubYE\n9DCb/m0hv0q1HeQWuNVrnw3BSQS/dHXSYxKnYfyJPQT0DBbpp5NhUmMJxK87kREr\n2AYatB4Knh7py2liqOinRSqHAntUv6cUtd1QzokeC4WNOWP7mNNRn3D5E8PFaP/W\nKQIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 16:08:07,209 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA7BKn84ehzrpBhCqWtF+X\nbX2WFcuURuPz4847Y0DSmQNGGp0d396Il6zyEXy6wJ6c7nPiPrhw8o6+FWmvbxsG\nLxqQu3d1tUre/VRaYKuX+l1LWXm0jAXSPfqva1wENgAB1itE9uEENR0/63JOAFFa\no+B/ln22z+BfBDOuJSEEkcDwmbx74W3IiaXV3HA4o7WJo31L4feb1Iw79uzOubYE\n9DCb/m0hv0q1HeQWuNVrnw3BSQS/dHXSYxKnYfyJPQT0DBbpp5NhUmMJxK87kREr\n2AYatB4Knh7py2liqOinRSqHAntUv6cUtd1QzokeC4WNOWP7mNNRn3D5E8PFaP/W\nKQIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:08:07,218 DEBUG core.py Correct username, starting passwd check.
2021-03-10 16:08:07,221 DEBUG core.py Auth message = {'response': 511, 'bin': '760a73b8e2275e396911f8d0a933b092365961278baacc40de06df8659fd80faf11e96644e127f58e3cfc11df9ae3d0c091813b1ad584874dab2e5bd37f037a9'}
2021-03-10 16:08:07,228 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53966)>, {'response': 511, 'bin': '760a73b8e2275e396911f8d0a933b092365961278baacc40de06df8659fd80faf11e96644e127f58e3cfc11df9ae3d0c091813b1ad584874dab2e5bd37f037a9'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:08:07,233 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53966)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:08:07,236 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53966)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:08:07,375 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53967)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:08:07,376 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615381686.5282617, 'user': {'account_name': 'test3', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA35NgXpKoG3Iz+3h5OHcL\n6bC5+HRchqrVhqlaOyS03vzMlRpHKteeGIr2kOt1CNhxqggQChzXA8SZErBu67Ur\nzEcgw8IHN6yVzr9W18E6olu1DuRQKzHY7Vhkv0uWS19Jnp9EU0PmR0sxBx7ZcWkt\n6WnMWBfsxBSMoDMUf5/b1BPOlTfbZ4TuY+jeoruM4ngFSmuT8T+6lfux4CkrdAn+\nCdleVV5el8ntT6SMdUDswLsHxZYxNmECSFL8fwbOHDqsu7H2RiYR+fMSsSORON7Y\nFMsBWRmzJ4bJ+/SNkg2tt7GqQb99fRCJt9x9OO3Bl2MQ9Ha5ah9QgUJ9twmFBAne\nyQIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 16:08:07,376 DEBUG core.py Start auth process for {'account_name': 'test3', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA35NgXpKoG3Iz+3h5OHcL\n6bC5+HRchqrVhqlaOyS03vzMlRpHKteeGIr2kOt1CNhxqggQChzXA8SZErBu67Ur\nzEcgw8IHN6yVzr9W18E6olu1DuRQKzHY7Vhkv0uWS19Jnp9EU0PmR0sxBx7ZcWkt\n6WnMWBfsxBSMoDMUf5/b1BPOlTfbZ4TuY+jeoruM4ngFSmuT8T+6lfux4CkrdAn+\nCdleVV5el8ntT6SMdUDswLsHxZYxNmECSFL8fwbOHDqsu7H2RiYR+fMSsSORON7Y\nFMsBWRmzJ4bJ+/SNkg2tt7GqQb99fRCJt9x9OO3Bl2MQ9Ha5ah9QgUJ9twmFBAne\nyQIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:08:07,384 DEBUG core.py Correct username, starting passwd check.
2021-03-10 16:08:07,387 DEBUG core.py Auth message = {'response': 511, 'bin': 'cbd416c60ac5b3a0e96ce75ce7225ed4867c469430c41be135731e4812a74a5782bc1cbf17aeef0c5a7cd377298d4629e4d06381639ec89ccf38f249ba5a95dd'}
2021-03-10 16:08:07,396 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53967)>, {'response': 511, 'bin': 'cbd416c60ac5b3a0e96ce75ce7225ed4867c469430c41be135731e4812a74a5782bc1cbf17aeef0c5a7cd377298d4629e4d06381639ec89ccf38f249ba5a95dd'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:08:07,407 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53967)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:08:07,412 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53967)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:08:08,031 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53965)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:08:08,031 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615381687.0280433, 'account_name': 'test2'}
2021-03-10 16:08:08,041 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53965)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3', 'Radif']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:08:08,045 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53966)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:08:08,045 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615381687.2351985, 'account_name': 'test1'}
2021-03-10 16:08:08,054 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53966)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3', 'Radif']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:08:08,058 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53967)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:08:08,058 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615381687.411646, 'account_name': 'test3'}
2021-03-10 16:08:08,067 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53967)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3', 'Radif']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:08:08,570 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53965)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:08:08,570 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615381688.2994514, 'user': 'test2'}
2021-03-10 16:08:08,584 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53965)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:08:08,591 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53966)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:08:08,591 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615381688.3366916, 'user': 'test1'}
2021-03-10 16:08:08,620 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53966)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:08:08,622 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53967)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:08:08,622 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615381688.3476913, 'user': 'test3'}
2021-03-10 16:08:08,629 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1324, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53967)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:08:36,154 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53966)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:08:36,154 DEBUG core.py Разбор сообщения от клиента : {'action': 'add', 'time': 1615381715.852412, 'user': 'test1', 'account_name': 'Radif'}
2021-03-10 16:08:56,674 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:08:56,674 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2021-03-10 16:08:56,687 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:08:56,688 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:08:56,929 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:09:21,390 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53971)
2021-03-10 16:09:21,392 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53972)
2021-03-10 16:09:21,414 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53973)
2021-03-10 16:09:21,920 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1296, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53971)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:09:21,920 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615381761.434159, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAkCloVMkGdxuoqW9r4GhP\np2rX5T0GDTIzNLc4fX38sGK9jUlQRvcbYrObBLfSFUQGy1fSQu+POR61Ik99BUpR\nXwzRVnnLXCFacoQDVkBZjNH1X0uff/Px7hOhZ1yXvM9aYercGvkC71oubZfJApDk\ny/0axjjxXToKUYLvwtXC8/s/wuToNKg9oI/EUtuIvMe/1pa7KBj322B92X+zijvD\nFW+YENwoyeykVARchsGZmtREKWJ/wTowHS8RY8nYhnoV6pW159H6RvPT6fyTf/GO\nGrMnmFkxDhs03eZQEElpjvItiJOysn/uirZw0kMlENTIO0kXhMq2t+nJwrL7YYpN\n6QIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 16:09:21,920 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAkCloVMkGdxuoqW9r4GhP\np2rX5T0GDTIzNLc4fX38sGK9jUlQRvcbYrObBLfSFUQGy1fSQu+POR61Ik99BUpR\nXwzRVnnLXCFacoQDVkBZjNH1X0uff/Px7hOhZ1yXvM9aYercGvkC71oubZfJApDk\ny/0axjjxXToKUYLvwtXC8/s/wuToNKg9oI/EUtuIvMe/1pa7KBj322B92X+zijvD\nFW+YENwoyeykVARchsGZmtREKWJ/wTowHS8RY8nYhnoV6pW159H6RvPT6fyTf/GO\nGrMnmFkxDhs03eZQEElpjvItiJOysn/uirZw0kMlENTIO0kXhMq2t+nJwrL7YYpN\n6QIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:09:21,927 DEBUG core.py Correct username, starting passwd check.
2021-03-10 16:09:21,930 DEBUG core.py Auth message = {'response': 511, 'bin': 'fadbc481d89a7af70564aca1598509dd16915fbc692e89189971fe1efda662e557333fbaf580a7362d5ab9e6d360216f65ad010fff88f62db340cf0db27b1880'}
2021-03-10 16:09:21,934 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1296, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53971)>, {'response': 511, 'bin': 'fadbc481d89a7af70564aca1598509dd16915fbc692e89189971fe1efda662e557333fbaf580a7362d5ab9e6d360216f65ad010fff88f62db340cf0db27b1880'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:09:21,937 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1296, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53971)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:09:21,940 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1296, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53971)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:10:04,843 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:10:04,844 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2021-03-10 16:10:04,857 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:10:04,858 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:10:05,413 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:10:21,797 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53979)
2021-03-10 16:10:21,912 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53980)
2021-03-10 16:10:21,921 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:21,921 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615381821.8412714, 'user': {'account_name': 'test3', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA35NgXpKoG3Iz+3h5OHcL\n6bC5+HRchqrVhqlaOyS03vzMlRpHKteeGIr2kOt1CNhxqggQChzXA8SZErBu67Ur\nzEcgw8IHN6yVzr9W18E6olu1DuRQKzHY7Vhkv0uWS19Jnp9EU0PmR0sxBx7ZcWkt\n6WnMWBfsxBSMoDMUf5/b1BPOlTfbZ4TuY+jeoruM4ngFSmuT8T+6lfux4CkrdAn+\nCdleVV5el8ntT6SMdUDswLsHxZYxNmECSFL8fwbOHDqsu7H2RiYR+fMSsSORON7Y\nFMsBWRmzJ4bJ+/SNkg2tt7GqQb99fRCJt9x9OO3Bl2MQ9Ha5ah9QgUJ9twmFBAne\nyQIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 16:10:21,922 DEBUG core.py Start auth process for {'account_name': 'test3', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA35NgXpKoG3Iz+3h5OHcL\n6bC5+HRchqrVhqlaOyS03vzMlRpHKteeGIr2kOt1CNhxqggQChzXA8SZErBu67Ur\nzEcgw8IHN6yVzr9W18E6olu1DuRQKzHY7Vhkv0uWS19Jnp9EU0PmR0sxBx7ZcWkt\n6WnMWBfsxBSMoDMUf5/b1BPOlTfbZ4TuY+jeoruM4ngFSmuT8T+6lfux4CkrdAn+\nCdleVV5el8ntT6SMdUDswLsHxZYxNmECSFL8fwbOHDqsu7H2RiYR+fMSsSORON7Y\nFMsBWRmzJ4bJ+/SNkg2tt7GqQb99fRCJt9x9OO3Bl2MQ9Ha5ah9QgUJ9twmFBAne\nyQIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:10:21,929 DEBUG core.py Correct username, starting passwd check.
2021-03-10 16:10:21,933 DEBUG core.py Auth message = {'response': 511, 'bin': 'f7665983d69d2051fe5f0be7dcd89feb67cf59ec3ab51804d13e15a4dff9a4c6c09094f0f1f2f70614ae3978630283f50dbb28f58ddbf5b400e074aaeb4af63c'}
2021-03-10 16:10:21,937 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>, {'response': 511, 'bin': 'f7665983d69d2051fe5f0be7dcd89feb67cf59ec3ab51804d13e15a4dff9a4c6c09094f0f1f2f70614ae3978630283f50dbb28f58ddbf5b400e074aaeb4af63c'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:10:21,940 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:10:21,944 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:10:22,189 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53981)
2021-03-10 16:10:22,192 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:22,193 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615381821.9441311, 'account_name': 'test3'}
2021-03-10 16:10:22,203 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3', 'Radif']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:10:22,207 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:22,207 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615381821.9531255, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA7BKn84ehzrpBhCqWtF+X\nbX2WFcuURuPz4847Y0DSmQNGGp0d396Il6zyEXy6wJ6c7nPiPrhw8o6+FWmvbxsG\nLxqQu3d1tUre/VRaYKuX+l1LWXm0jAXSPfqva1wENgAB1itE9uEENR0/63JOAFFa\no+B/ln22z+BfBDOuJSEEkcDwmbx74W3IiaXV3HA4o7WJo31L4feb1Iw79uzOubYE\n9DCb/m0hv0q1HeQWuNVrnw3BSQS/dHXSYxKnYfyJPQT0DBbpp5NhUmMJxK87kREr\n2AYatB4Knh7py2liqOinRSqHAntUv6cUtd1QzokeC4WNOWP7mNNRn3D5E8PFaP/W\nKQIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 16:10:22,207 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA7BKn84ehzrpBhCqWtF+X\nbX2WFcuURuPz4847Y0DSmQNGGp0d396Il6zyEXy6wJ6c7nPiPrhw8o6+FWmvbxsG\nLxqQu3d1tUre/VRaYKuX+l1LWXm0jAXSPfqva1wENgAB1itE9uEENR0/63JOAFFa\no+B/ln22z+BfBDOuJSEEkcDwmbx74W3IiaXV3HA4o7WJo31L4feb1Iw79uzOubYE\n9DCb/m0hv0q1HeQWuNVrnw3BSQS/dHXSYxKnYfyJPQT0DBbpp5NhUmMJxK87kREr\n2AYatB4Knh7py2liqOinRSqHAntUv6cUtd1QzokeC4WNOWP7mNNRn3D5E8PFaP/W\nKQIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:10:22,213 DEBUG core.py Correct username, starting passwd check.
2021-03-10 16:10:22,217 DEBUG core.py Auth message = {'response': 511, 'bin': 'd078abe30a0e769091a0af685d1077ce356ed037c19fab6a8da749471a982056c32521140a0f5353c1f0842b878ef28b7c89ebe4114f74a3e4894ae98b65920e'}
2021-03-10 16:10:22,224 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>, {'response': 511, 'bin': 'd078abe30a0e769091a0af685d1077ce356ed037c19fab6a8da749471a982056c32521140a0f5353c1f0842b878ef28b7c89ebe4114f74a3e4894ae98b65920e'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:10:22,230 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:10:22,232 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:10:22,480 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1276, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53981)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:22,481 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615381821.9601219, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAkCloVMkGdxuoqW9r4GhP\np2rX5T0GDTIzNLc4fX38sGK9jUlQRvcbYrObBLfSFUQGy1fSQu+POR61Ik99BUpR\nXwzRVnnLXCFacoQDVkBZjNH1X0uff/Px7hOhZ1yXvM9aYercGvkC71oubZfJApDk\ny/0axjjxXToKUYLvwtXC8/s/wuToNKg9oI/EUtuIvMe/1pa7KBj322B92X+zijvD\nFW+YENwoyeykVARchsGZmtREKWJ/wTowHS8RY8nYhnoV6pW159H6RvPT6fyTf/GO\nGrMnmFkxDhs03eZQEElpjvItiJOysn/uirZw0kMlENTIO0kXhMq2t+nJwrL7YYpN\n6QIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 16:10:22,481 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAkCloVMkGdxuoqW9r4GhP\np2rX5T0GDTIzNLc4fX38sGK9jUlQRvcbYrObBLfSFUQGy1fSQu+POR61Ik99BUpR\nXwzRVnnLXCFacoQDVkBZjNH1X0uff/Px7hOhZ1yXvM9aYercGvkC71oubZfJApDk\ny/0axjjxXToKUYLvwtXC8/s/wuToNKg9oI/EUtuIvMe/1pa7KBj322B92X+zijvD\nFW+YENwoyeykVARchsGZmtREKWJ/wTowHS8RY8nYhnoV6pW159H6RvPT6fyTf/GO\nGrMnmFkxDhs03eZQEElpjvItiJOysn/uirZw0kMlENTIO0kXhMq2t+nJwrL7YYpN\n6QIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:10:22,489 DEBUG core.py Correct username, starting passwd check.
2021-03-10 16:10:22,493 DEBUG core.py Auth message = {'response': 511, 'bin': 'ecd5631fee85c6a556d8694c4a342142ea55e894b0b0ab1941f0cf08633ad28ac3bd5c3ce4c007e7f6aaccb253bb26bb7efcba096ec0a1274bfa7d07410257df'}
2021-03-10 16:10:22,501 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1276, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53981)>, {'response': 511, 'bin': 'ecd5631fee85c6a556d8694c4a342142ea55e894b0b0ab1941f0cf08633ad28ac3bd5c3ce4c007e7f6aaccb253bb26bb7efcba096ec0a1274bfa7d07410257df'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:10:22,509 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1276, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53981)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:10:22,513 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1276, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53981)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:10:23,148 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:23,148 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615381822.3958921, 'user': 'test3'}
2021-03-10 16:10:23,161 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:10:23,172 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:23,175 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615381822.2328446, 'account_name': 'test1'}
2021-03-10 16:10:23,200 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3', 'Radif']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:10:23,203 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1276, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53981)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:23,203 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615381822.512024, 'account_name': 'test2'}
2021-03-10 16:10:23,210 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1276, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53981)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3', 'Radif']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:10:23,713 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:23,713 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615381823.4048378, 'user': 'test1'}
2021-03-10 16:10:23,724 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:10:23,728 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1276, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53981)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:23,729 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615381823.462462, 'user': 'test2'}
2021-03-10 16:10:23,740 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1276, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53981)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:10:30,751 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:30,751 DEBUG core.py Разбор сообщения от клиента : {'action': 'add', 'time': 1615381830.6346636, 'user': 'test3', 'account_name': 'test1'}
2021-03-10 16:10:30,927 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:10:34,433 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:34,433 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1615381834.1169775, 'account_name': 'test1'}
2021-03-10 16:10:34,442 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA7BKn84ehzrpBhCqWtF+X\nbX2WFcuURuPz4847Y0DSmQNGGp0d396Il6zyEXy6wJ6c7nPiPrhw8o6+FWmvbxsG\nLxqQu3d1tUre/VRaYKuX+l1LWXm0jAXSPfqva1wENgAB1itE9uEENR0/63JOAFFa\no+B/ln22z+BfBDOuJSEEkcDwmbx74W3IiaXV3HA4o7WJo31L4feb1Iw79uzOubYE\n9DCb/m0hv0q1HeQWuNVrnw3BSQS/dHXSYxKnYfyJPQT0DBbpp5NhUmMJxK87kREr\n2AYatB4Knh7py2liqOinRSqHAntUv6cUtd1QzokeC4WNOWP7mNNRn3D5E8PFaP/W\nKQIDAQAB\n-----END PUBLIC KEY-----'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:10:39,949 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:39,949 DEBUG core.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test3', 'to': 'test1', 'time': 1615381839.3969653, 'mess_text': 'wvWKMusA/fJDn8PE88tSDa+F9CCPh58skNULfRgw2Y6Se+sSHdd3ti8OZZCVaZGUIkS7e+lkLkzQsB3kE9qtIKxnA6FN0C+OzK0xtgINqH46Qc0DmW5bUO+c1uRkODeaegpzkCgcGDRmtcXDoWSJpEtAf74haZ1xWa3alEun8UQSodCuTU8S+9F1/vTA15us0ZNnAVGE8dCTtDKtgpl7HpjnM0yb3W1d1KnmOYV/t57JegMIbRofT1jO5MWL01hdr4+61vB99g7XE7gG8mElMd4yt+YzFsXmNhPDvXvtJMG9MndLA950aOoL5v/VN5XDpoIIV6gF0VGRdhx/ZgrTpQ=='}
2021-03-10 16:10:40,187 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>, {'action': 'message', 'from': 'test3', 'to': 'test1', 'time': 1615381839.3969653, 'mess_text': 'wvWKMusA/fJDn8PE88tSDa+F9CCPh58skNULfRgw2Y6Se+sSHdd3ti8OZZCVaZGUIkS7e+lkLkzQsB3kE9qtIKxnA6FN0C+OzK0xtgINqH46Qc0DmW5bUO+c1uRkODeaegpzkCgcGDRmtcXDoWSJpEtAf74haZ1xWa3alEun8UQSodCuTU8S+9F1/vTA15us0ZNnAVGE8dCTtDKtgpl7HpjnM0yb3W1d1KnmOYV/t57JegMIbRofT1jO5MWL01hdr4+61vB99g7XE7gG8mElMd4yt+YzFsXmNhPDvXvtJMG9MndLA950aOoL5v/VN5XDpoIIV6gF0VGRdhx/ZgrTpQ=='}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_message
2021-03-10 16:10:40,187 INFO core.py Отправлено сообщение пользователю test1 от пользователя test3.
2021-03-10 16:10:40,194 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:10:48,704 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:48,704 DEBUG core.py Разбор сообщения от клиента : {'action': 'add', 'time': 1615381848.2493567, 'user': 'test1', 'account_name': 'test3'}
2021-03-10 16:10:48,917 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:10:50,422 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:50,423 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1615381850.3107953, 'account_name': 'test3'}
2021-03-10 16:10:50,430 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA35NgXpKoG3Iz+3h5OHcL\n6bC5+HRchqrVhqlaOyS03vzMlRpHKteeGIr2kOt1CNhxqggQChzXA8SZErBu67Ur\nzEcgw8IHN6yVzr9W18E6olu1DuRQKzHY7Vhkv0uWS19Jnp9EU0PmR0sxBx7ZcWkt\n6WnMWBfsxBSMoDMUf5/b1BPOlTfbZ4TuY+jeoruM4ngFSmuT8T+6lfux4CkrdAn+\nCdleVV5el8ntT6SMdUDswLsHxZYxNmECSFL8fwbOHDqsu7H2RiYR+fMSsSORON7Y\nFMsBWRmzJ4bJ+/SNkg2tt7GqQb99fRCJt9x9OO3Bl2MQ9Ha5ah9QgUJ9twmFBAne\nyQIDAQAB\n-----END PUBLIC KEY-----'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:10:56,938 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:10:56,939 DEBUG core.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test3', 'time': 1615381856.6280885, 'mess_text': 'OhwrIbblyS37xgFJSOyLZjJSdKeOnJ2mbwvZL4n0jzk3bBd+0j1fMv4JVdwKvJ9EOMiY2S2QmUoebTfC0xGmZxuA4SEH6uJYdTX7+tcn+uh+bTXP9ZQ6JBGj/+hiDk54DVjI3x9BYyHzaguBwB7ZIaYSKcNE3zbzHp/8aav6+vTd4R+TUBYE2wRM7XzK46GeiiB1pjf8FoSFlG3Iox+jsiBqDTjKmuF1AZm5UV4pbzLGkyOBz0qp3wx1HHsJMasmIIoK+NxEkMHnckfNAqEfrTnv5iH0sUGJdyhQ4rPjJ5I8gufwguoVt5x55hxhVL0exxN0NBb0/f5khMAFSfGwnw=='}
2021-03-10 16:10:57,105 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>, {'action': 'message', 'from': 'test1', 'to': 'test3', 'time': 1615381856.6280885, 'mess_text': 'OhwrIbblyS37xgFJSOyLZjJSdKeOnJ2mbwvZL4n0jzk3bBd+0j1fMv4JVdwKvJ9EOMiY2S2QmUoebTfC0xGmZxuA4SEH6uJYdTX7+tcn+uh+bTXP9ZQ6JBGj/+hiDk54DVjI3x9BYyHzaguBwB7ZIaYSKcNE3zbzHp/8aav6+vTd4R+TUBYE2wRM7XzK46GeiiB1pjf8FoSFlG3Iox+jsiBqDTjKmuF1AZm5UV4pbzLGkyOBz0qp3wx1HHsJMasmIIoK+NxEkMHnckfNAqEfrTnv5iH0sUGJdyhQ4rPjJ5I8gufwguoVt5x55hxhVL0exxN0NBb0/f5khMAFSfGwnw=='}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_message
2021-03-10 16:10:57,105 INFO core.py Отправлено сообщение пользователю test3 от пользователя test1.
2021-03-10 16:10:57,110 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53980)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:11:37,427 INFO core.py Установлено соедение с ПК ('127.0.0.1', 53987)
2021-03-10 16:11:37,931 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:11:37,932 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1615381897.458188, 'user': {'account_name': 'Radif', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAnI4oYxe6Cy5uKvDezZIn\nBzEs0TjoORJe9zVSB3aSHsCQKtxK9W6sJR3BMvVP5bwpeRLh6rzPTbvuSCT165/l\nR06EfJGwa5+P5DO8hADKUN61VPiZZdIBxEslWHMGxqPjRX+bC25kxaDEtqRnOX6J\nqSECmKsY1zIQWDzwqL3C7aXAUYTsVcxJM4/uD0OmK/tskV/wxT4Q9j9OrXKUgNHC\nD4C2H9+5/qCtZgCad0xrk1NzN6ATmE98eguJALbsAuin3oeaQT6w9b7ak0hxf2cC\nqe/8KilobpOqBbWw7DZqnOCOhzagjVasiLWvNTstyY8Gl7Xjj0sxa00IKgILqtLc\nJwIDAQAB\n-----END PUBLIC KEY-----'}}
2021-03-10 16:11:37,932 DEBUG core.py Start auth process for {'account_name': 'Radif', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAnI4oYxe6Cy5uKvDezZIn\nBzEs0TjoORJe9zVSB3aSHsCQKtxK9W6sJR3BMvVP5bwpeRLh6rzPTbvuSCT165/l\nR06EfJGwa5+P5DO8hADKUN61VPiZZdIBxEslWHMGxqPjRX+bC25kxaDEtqRnOX6J\nqSECmKsY1zIQWDzwqL3C7aXAUYTsVcxJM4/uD0OmK/tskV/wxT4Q9j9OrXKUgNHC\nD4C2H9+5/qCtZgCad0xrk1NzN6ATmE98eguJALbsAuin3oeaQT6w9b7ak0hxf2cC\nqe/8KilobpOqBbWw7DZqnOCOhzagjVasiLWvNTstyY8Gl7Xjj0sxa00IKgILqtLc\nJwIDAQAB\n-----END PUBLIC KEY-----'}
2021-03-10 16:11:37,937 DEBUG core.py Correct username, starting passwd check.
2021-03-10 16:11:37,941 DEBUG core.py Auth message = {'response': 511, 'bin': '9c0770ab3c388868a9cfb35ff09619861280adb37cdee062860c35310fc92f834622d45a6e9766b6f4532c2281b874caa7fb8633ff5047d60c1bcc11d97cc325'}
2021-03-10 16:11:37,948 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>, {'response': 511, 'bin': '9c0770ab3c388868a9cfb35ff09619861280adb37cdee062860c35310fc92f834622d45a6e9766b6f4532c2281b874caa7fb8633ff5047d60c1bcc11d97cc325'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:11:37,952 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:11:37,955 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции autorize_user
2021-03-10 16:11:38,703 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:11:38,704 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1615381897.955565, 'account_name': 'Radif'}
2021-03-10 16:11:38,710 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3', 'Radif']}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:11:39,214 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:11:39,214 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1615381898.9401262, 'user': 'Radif'}
2021-03-10 16:11:39,226 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:11:44,234 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:11:44,235 DEBUG core.py Разбор сообщения от клиента : {'action': 'add', 'time': 1615381903.8981788, 'user': 'Radif', 'account_name': 'test3'}
2021-03-10 16:11:44,393 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:11:49,401 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:11:49,401 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1615381908.986302, 'account_name': 'test3'}
2021-03-10 16:11:49,410 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA35NgXpKoG3Iz+3h5OHcL\n6bC5+HRchqrVhqlaOyS03vzMlRpHKteeGIr2kOt1CNhxqggQChzXA8SZErBu67Ur\nzEcgw8IHN6yVzr9W18E6olu1DuRQKzHY7Vhkv0uWS19Jnp9EU0PmR0sxBx7ZcWkt\n6WnMWBfsxBSMoDMUf5/b1BPOlTfbZ4TuY+jeoruM4ngFSmuT8T+6lfux4CkrdAn+\nCdleVV5el8ntT6SMdUDswLsHxZYxNmECSFL8fwbOHDqsu7H2RiYR+fMSsSORON7Y\nFMsBWRmzJ4bJ+/SNkg2tt7GqQb99fRCJt9x9OO3Bl2MQ9Ha5ah9QgUJ9twmFBAne\nyQIDAQAB\n-----END PUBLIC KEY-----'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:12:10,933 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:12:10,934 DEBUG core.py Разбор сообщения от клиента : {'action': 'message', 'from': 'Radif', 'to': 'test3', 'time': 1615381930.715771, 'mess_text': 'AdXRCt5xNp/j/nxI6kdXUHMmPZGBS/Jk93EVM4bRlGBoKc93KLDS0dbkcICa5yfJTc4k+OFL9YfGSCEQeQnrRuWRGoPbGf0Bbbwe3BtmKfq8Nns3dni3QdKQh97WEsSVkWDfO67Wjk7WOOjQi/WzY5r8OJQMiDS2lCGQ6Ib/z3Vt1t+wB0P9RYVRmIYFESUQVIjG39ErjpVIsiHhW814HWrPaiDnOgE+zJhcCBTvkFznlXPS3hRdalf6G5h/6r2GWJ8EgNU1Ve6aZOrRxXV1sjciVY7i2hx03gd+r87K5a4wFdi4vaT8xw3QLxRdPqdMYwXz72h3tabFpZ0LYbVA5Q=='}
2021-03-10 16:12:11,141 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>, {'action': 'message', 'from': 'Radif', 'to': 'test3', 'time': 1615381930.715771, 'mess_text': 'AdXRCt5xNp/j/nxI6kdXUHMmPZGBS/Jk93EVM4bRlGBoKc93KLDS0dbkcICa5yfJTc4k+OFL9YfGSCEQeQnrRuWRGoPbGf0Bbbwe3BtmKfq8Nns3dni3QdKQh97WEsSVkWDfO67Wjk7WOOjQi/WzY5r8OJQMiDS2lCGQ6Ib/z3Vt1t+wB0P9RYVRmIYFESUQVIjG39ErjpVIsiHhW814HWrPaiDnOgE+zJhcCBTvkFznlXPS3hRdalf6G5h/6r2GWJ8EgNU1Ve6aZOrRxXV1sjciVY7i2hx03gd+r87K5a4wFdi4vaT8xw3QLxRdPqdMYwXz72h3tabFpZ0LYbVA5Q=='}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_message
2021-03-10 16:12:11,141 INFO core.py Отправлено сообщение пользователю test3 от пользователя Radif.
2021-03-10 16:12:11,147 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:12:18,656 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:12:18,656 DEBUG core.py Разбор сообщения от клиента : {'action': 'add', 'time': 1615381938.3929057, 'user': 'test3', 'account_name': 'Radif'}
2021-03-10 16:12:18,844 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:12:20,349 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:12:20,350 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1615381940.343931, 'account_name': 'Radif'}
2021-03-10 16:12:20,361 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAnI4oYxe6Cy5uKvDezZIn\nBzEs0TjoORJe9zVSB3aSHsCQKtxK9W6sJR3BMvVP5bwpeRLh6rzPTbvuSCT165/l\nR06EfJGwa5+P5DO8hADKUN61VPiZZdIBxEslWHMGxqPjRX+bC25kxaDEtqRnOX6J\nqSECmKsY1zIQWDzwqL3C7aXAUYTsVcxJM4/uD0OmK/tskV/wxT4Q9j9OrXKUgNHC\nD4C2H9+5/qCtZgCad0xrk1NzN6ATmE98eguJALbsAuin3oeaQT6w9b7ak0hxf2cC\nqe/8KilobpOqBbWw7DZqnOCOhzagjVasiLWvNTstyY8Gl7Xjj0sxa00IKgILqtLc\nJwIDAQAB\n-----END PUBLIC KEY-----'}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:12:26,369 DEBUG core.py Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>,), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции run
2021-03-10 16:12:26,370 DEBUG core.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test3', 'to': 'Radif', 'time': 1615381946.3152328, 'mess_text': 'mLjDUrqKrrukbYGPVk27QrR3jQaXwQ4KWzEY21ncF5RoGlKJKVq0D05L1d8UoXZe/2e1DuBXhscwYtys9ZRrv8wVtD/TO3K+ewjBEguN9p9rGk5Z/P9bgW6dFVF3FwU3GXLQ8xuY0u4vvVeQVbajiOv4HMnZuzz12orrpOEfPgdv5qGYEtzJm5IuUz+IU9vm3aMA8viAtxcld+JdtTkcf00DyAmjDF4gv6d06YXNJxs55ovxZ1rlNAf2vhw9mSVSW4SEQZv2MClZhWGjKvpiM7zpy1DjoGqNa9chyS6Vj+zt9v69BZ/MYmroxyAU2FvlSOn25onMqMadpC36vKtxyw=='}
2021-03-10 16:12:26,525 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53987)>, {'action': 'message', 'from': 'test3', 'to': 'Radif', 'time': 1615381946.3152328, 'mess_text': 'mLjDUrqKrrukbYGPVk27QrR3jQaXwQ4KWzEY21ncF5RoGlKJKVq0D05L1d8UoXZe/2e1DuBXhscwYtys9ZRrv8wVtD/TO3K+ewjBEguN9p9rGk5Z/P9bgW6dFVF3FwU3GXLQ8xuY0u4vvVeQVbajiOv4HMnZuzz12orrpOEfPgdv5qGYEtzJm5IuUz+IU9vm3aMA8viAtxcld+JdtTkcf00DyAmjDF4gv6d06YXNJxs55ovxZ1rlNAf2vhw9mSVSW4SEQZv2MClZhWGjKvpiM7zpy1DjoGqNa9chyS6Vj+zt9v69BZ/MYmroxyAU2FvlSOn25onMqMadpC36vKtxyw=='}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_message
2021-03-10 16:12:26,525 INFO core.py Отправлено сообщение пользователю Radif от пользователя test3.
2021-03-10 16:12:26,529 DEBUG core.py Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 53979)>, {'response': 200}), {}. Вызов из модуля common.utils.Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2021-03-10 16:37:13,352 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:37:13,352 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:37:13,364 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:37:13,365 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:37:13,582 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:39:54,849 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 16:40:20,912 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:40:20,913 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:40:20,926 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:40:20,927 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:40:21,162 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:40:27,350 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 16:40:29,366 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:40:29,366 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:40:29,380 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:40:29,381 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:40:29,611 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:40:35,661 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 16:40:51,690 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:40:51,691 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:40:51,706 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:40:51,708 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:40:51,937 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:40:54,534 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 16:41:04,156 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:41:04,156 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:41:04,169 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:41:04,170 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:41:04,385 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:41:07,524 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 16:41:54,849 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:41:54,850 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:41:54,862 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:41:54,863 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:41:55,151 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:43:47,711 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 16:43:51,794 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:43:51,794 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:43:51,806 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:43:51,807 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:43:52,119 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:44:19,409 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:44:19,409 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:44:19,422 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:44:19,423 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:44:19,714 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:45:06,823 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:45:06,823 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:45:06,846 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:45:06,848 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:45:07,214 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:45:12,566 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 16:45:29,794 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:45:29,794 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:45:29,806 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:45:29,807 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:45:30,083 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:45:36,281 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 16:46:10,910 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:46:10,910 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:46:10,926 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:46:10,927 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:46:11,213 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:47:00,110 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:47:00,110 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:47:00,137 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:47:00,139 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:47:00,523 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:47:03,275 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 16:47:05,323 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:47:05,323 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:47:05,336 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:47:05,337 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:47:05,705 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:47:12,296 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 16:51:34,212 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:51:34,212 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:51:34,225 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:51:34,226 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:51:34,637 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:51:42,931 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 16:52:21,406 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:52:21,406 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:52:21,427 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:52:21,428 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:52:22,092 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:52:24,749 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:52:24,749 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:52:24,831 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:52:24,832 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:52:26,131 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:52:31,418 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 16:52:57,430 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:52:57,431 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:52:57,446 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:52:57,447 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:52:57,760 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:53:57,772 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:53:57,773 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:53:57,794 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:53:57,796 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:53:58,220 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:56:18,170 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:56:18,170 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:56:18,208 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:56:18,210 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:56:18,609 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:57:03,220 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:57:03,221 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:57:03,244 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:57:03,246 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:57:03,671 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:57:09,763 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 16:57:28,305 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:57:28,305 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 16:57:28,318 DEBUG server.py Аргументы успешно загружены.
2021-03-10 16:57:28,320 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 16:57:28,592 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 16:57:30,630 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 17:01:37,204 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 17:01:37,204 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 17:01:37,216 DEBUG server.py Аргументы успешно загружены.
2021-03-10 17:01:37,217 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 17:01:37,449 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 17:01:40,038 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
2021-03-10 17:01:59,899 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 17:01:59,899 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 17:01:59,913 DEBUG server.py Аргументы успешно загружены.
2021-03-10 17:01:59,915 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 17:02:00,178 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 17:02:59,853 DEBUG server.py Была вызвана функция config_load c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 17:02:59,853 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['D:/YandexDisk/GeekBrains/Факультет Python-разработки/2.6 Базы данных и PyQT 18.01.2021 MSK (UTC+3)/Databases_ans_PyQt/lesson06/server.py']
2021-03-10 17:02:59,871 DEBUG server.py Аргументы успешно загружены.
2021-03-10 17:02:59,873 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции main
2021-03-10 17:03:00,155 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-03-10 17:03:02,221 DEBUG server.py Была вызвана функция main c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main().Вызов из функции <module>
